src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -325,4 +326,37 @@ avoiding peak traffic in order to address both inconsistencies diagnostic and fi
 
 #### How to solve
 
-Under development: Task for solving mailbox inconsistencies
\ No newline at end of file
+Under development: Task for solving mailbox inconsistencies
+
+## Mail Queue
+
+An email queue is a mandatory component of SMTP servers. It is a system that creates an array of emails that are waiting to be processed for delivery. Email queuing is a form of Message Queuing – an asynchronous service-to-service communication. A message queue is meant to decouple a producing process from a consuming one. An email queue decouples the sender from the recipient. It allows them to communicate without being connected. As such, the queued emails wait for processing until the recipient is available to receive them. As James is an Email Server, it also supports mail queue as well.
+
```

Comment:
A bit of explanation on the architecture can be helpful:

```
The mailqueue is a complex components reling on the capability of several third party softwares:
 - RabbitMQ ensures messaging function, and avoids polling
 - Cassandra enables administrative operations (browse + deletes see section below) using  a time serie which might require fine performance tuning (see http://... )
 - Object Storage stores potentially large binary payload
```

Reply:
It's not "time serie" but "time series"